src/resolve/resolved-dnssd.c

```diff
@@ -253,6 +262,16 @@ int dnssd_update_rrs(DnssdService *s) {
         if (!s->ptr_rr->ptr.name)
                 goto oom;
 
+        s->sub_ptr_rr = dns_resource_record_new_full(DNS_CLASS_IN, DNS_TYPE_PTR,
+                                                 selective_name);
```

Comment:
we break lines differently btw. i.e. line width is 109ch. if you break, break after the `(` of the function call, and indent to twice the usual indentation (i.e. 16 spaces)